web/README.md

```diff
@@ -0,0 +1,86 @@
+# ARC TRE portal frontent
+
+This React + Typescript project was set up using [Vite](https://vitejs.dev/guide/why.html).
+
+Development files are located in '/src' with all components routed into App.ts.
+
+---
+
+# Requirements
+
+Make sure you have read through the `README.md` file at the PROJECT_ROOT directory. This will give you more background on the application architecture.
```

Comment:
```suggestion
Make sure you have read through the root [`README.md`](../README.md) file. This will give you more background on the application architecture.
```